yt/docker/ytsaurus/Dockerfile

```diff
@@ -4,39 +4,75 @@
 ARG BASE_REPOSITORY="ghcr.io/ytsaurus/ytsaurus-nightly"
 ARG BASE_IMAGE="latest"
 
+# Args for odin.
+ARG PYTHON_RUNTIME_VERSION="3.12"
+
+ARG ODIN_ROOT="/ytsaurus"
+ARG ODIN_VIRTUAL_ENV="${ODIN_ROOT}/virtualenv"
+ARG ODIN_RUNTIME_ROOT="/var/odin"
+ARG ODIN_CHECKS_DIR="${ODIN_RUNTIME_ROOT}/checks"
+ARG ODIN_CHECKS_DATA_DIR="${ODIN_RUNTIME_ROOT}/checks-data"
+
+# Flow control args.
 ARG PYTHON_BUILD_BASE="copy-ytsaurus-python-binaries"
+ARG ROOT_IMAGE="ubuntu-base"
+
+##########################################################################################
+
+FROM mirror.gcr.io/ubuntu:focal AS ubuntu-base
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y \
+      python3-pip \
+      python3.8  \
+      python3.8-distutils \
```

Comment:
Why we install python3.8 here?

Reply:
Isn't this our "default" system python that we have had for a while now?